docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md

```diff
@@ -388,7 +389,21 @@ As an API service developer, you set multiple configuration settings in your app
       * `metadata-map.routed-services.<prefix>.serviceUrl`
 
           Both gateway-url and service-url parameters specify how the API service endpoints are mapped to the API gateway endpoints. The service-url parameter points to the target endpoint on the gateway.
-
+      * `eureka.instance.metadata-map.apiml.apiInfo.apiId`
+      
+          Specifies the API identifier that is registered in the API Mediation Layer installation. The API ID uniquely identifies the API in the API Mediation Layer.
+          The same API can be provided by multiple service. The API ID can be used to locate same APIs that are provided by different services. The creator of the API defines this ID.
+          The API ID needs to be string up to 64 characters that is using lowercase alphanumeric characters and a dot: `.`. It is recommended to use your organization as the prefix.
+      * `eureka.instance.metadata-map.apiml.apiInfo.gatewayUrl`
+      
+          The base path at the API gateway where the API is available. It should bethe same as a _gatewayUrl_ value in the _routes_ sections.
+          
+      * `eureka.instance.metadata-map.apiml.apiInfo.documentationUrl`
+      
+          (Optional) Link to external documentation, if needed. Can coexist with Swagger documentation
+      * `eureka.instance.metadata-map.apiml.apiInfo.swaggerUrl`
+      
+          (Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document that provides the API documentation for this API is available.                   
```

Comment:
This is unclear. Is it clear enough to simply say,
"(Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document is available. "

Reply:
This was a copy from _docs/extend/extend-apiml/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md. 
I simplified it everywhere in the code.